rclcpp/include/rclcpp/parameter_client.hpp

```diff
@@ -241,6 +253,12 @@ class AsyncParametersClient
   rclcpp::client::Client<rcl_interfaces::srv::DescribeParameters>::SharedPtr
     describe_parameters_client_;
   std::string remote_node_name_;
+
+  std::unordered_set<std::shared_ptr<std::promise<std::vector<rclcpp::parameter::ParameterVariant>>>> get_parameters_promises_;
+  std::unordered_set<std::shared_ptr<std::promise<std::vector<rclcpp::parameter::ParameterType>>>> get_parameter_types_promises_;
+  std::unordered_set<std::shared_ptr<std::promise<std::vector<rcl_interfaces::msg::SetParametersResult>>>> set_parameters_promises_;
+  std::unordered_set<std::shared_ptr<std::promise<rcl_interfaces::msg::SetParametersResult>>> set_parameters_atomically_promises_;
+  std::unordered_set<std::shared_ptr<std::promise<rcl_interfaces::msg::ListParametersResult>>> list_parameters_promises_;
```

Comment:
Can you break these into multi line statements? These lines are over 120 characters long.